src/utils/AppConsts.js

```diff
@@ -0,0 +1,11 @@
+/*
+ * @flow
+ */
+
+const AUTH0_CLIENT_ID = 'KvwsETaUxuXVjW2cmz2LbdqXQBdYs6wH';
+const AUTH0_DOMAIN = 'loom.auth0.com';
+
+module.exports = {
```

Comment:
`module.exports` is for NodeJS. you can use standard ES6 export syntax for any of the app source code.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export